libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1728,3 +1723,40 @@ def generate_and_compare_both_version(input_installing_version, installed_versio
         else:
             LOGGER.info("Installing version is not higher than installed version.")
         return installing_version
+
+    def update_sol_for_granular_deploy(self, file_path: str, host_list: list,
+                                       image: str, deployment_type: str) -> tuple:
+        """
+        Helper function to update image in solution.yaml.
+        :param: file_path: Filename with complete path
+        :param: host_list: List of setup hosts
+        :param: image: Image to be used for deployment
+        :param: deployment_type: Type of deployment(Standard/Data-Only)
+        :return: True/False and local file
+        """
+        LOGGER.debug("Update nodes section with setup details.")
+        resp = self.update_nodes_sol_file(file_path, host_list)
+        if not resp[0]:
+            return False, "solution.yaml is not updated properly."
+        LOGGER.debug("Update storage section and deployment type.")
+        with open(file_path) as soln:
+            conf = yaml.safe_load(soln)
+            parent_key = conf['solution']
+            storage_key = parent_key["storage"]
+            cvg_key = storage_key["cvg1"]["devices"]["data"]
```

Comment:
use variable instead of hardcoding the cvg

Reply:
Done.